### DISCUSSION TITLE
Bug: Error 403

### DISCUSSION THREAD

**Discussion by Icelagy** [eritislami/evobot#302] [Q&A]
I get this error when using "Play" command:

{
  code: 403,
  message: 'The request cannot be completed because you have exceeded your <a href="/youtube/v3/getting-started#quota">quota</a>.',
  errors: [
    {
      message: 'The request cannot be completed because you have exceeded your <a href="/youtube/v3/getting-started#quota">quota</a>.',
      domain: 'youtube.quota',
      reason: 'quotaExceeded'
    }
  ]
}

**Comment by aka-pal** [comment]
Change YouTube api key

**Comment by Icelagy** [comment]
I already changed it, twice.

**Reply by dcapape** [reply]
Did you got it? I'm having the same issue with an API key without units used.

**Comment by wolf-yuan-6115** [comment]
Try to reinstall module

**Comment by Icelagy** [comment]
How do I reinstall modules.

**Comment by eritislami** [comment]
Not a bug, moving to discussions

**Comment by wolf-yuan-6115** [comment]
Oh I know, you have exceeded your YouTube API key limit (10000 requests), just wait.
